releaseNotes/JavaScriptFeatureManagementApplicationInsightsBrowser.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft Feature Management for JavaScript
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 2.0.0-preview.1 - Oct 15, 2024
+
+* Added a built-in method to create telemetry publisher with Application Insights.
+
+* Added a built-in `TrackEvent` method to send custom event to Application Insights with targeting id attached.
```

Comment:
Does this make sense?

Introduced a `TrackEvent` API, allowing users to replace existing Application Insights `TrackEvent` calls to include targeting information in custom events sent to Application Insights.

Reply:
Our function signature is different from the Application Insights' `TrackEvent`. 
appInsightsClient.TrackEvent("EventName") vs TrackEvent(appInsightsClient, targetingId, "EventName")

So my concern is that whether it will be misleading.

Reply:
Isn't the reason you name the new API `TrackEvent` that you want users to use the new API to replace the existing `TrackEvent`? We didn't say they have the same signature.

Reply:
Ok, then it wil be fine.